torchrl/trainers/helpers/collectors.py

```diff
@@ -241,7 +241,7 @@ def _make_collector(
 def make_collector_offpolicy(
     make_env: Callable[[], _EnvClass],
     actor_model_explore: Union[TensorDictModuleWrapper, ProbabilisticTensorDictModule],
-    args: Namespace,
+    args: DictConfig,
```

Comment:
same here, could we update the variable names to be `cfg` to make it more consistent throughout?